docs/aws/README.md

```diff
@@ -3,31 +3,32 @@
 This guide will provide a complete Karpenter installation for AWS.
 These steps are opinionated and may need to be adapted for your use case.
 
+> This guide should take less than 1 hour to complete and cost less than $.25
```

Comment:
Is this $.25 one off (because it'll all be shut down later?) or $.25 per some unit of time (continuously?)

Reply:
$.25 one time to go through the complete tutorial. I ran it in my personal account yesterday which only had an existing $.01 ECR charge.

![image](https://user-images.githubusercontent.com/371796/123846150-da741980-d8c9-11eb-8df6-f6d2d2bec39f.png)

running through this tutorial and taking my time between some steps cost $.24